### ISSUE TITLE
[nativescript-plaid] Link is crashing at the very last step

### ISSUE THREAD

**Issue by Benoye** [nstudio/nativescript-plugins#26]
Using the nativescript-plaid plugin, it crashes at the very end of the process when clicking on the "Continue" button
I am using plaid in sandbox mode

Only logs I have is a SIGABRT and I struggle to debug this error..

iOS : tested on 14.1 and 14.4
Plugin version : 1.0.4
NS version : 7.2.0
XCode : 12.4

**Comment by Benoye** [comment]
Note that it does not crash on Android
(tested on Android 11.0)

**Comment by davecoffin** [comment]
Paste all the logs when the crash occurs, you didn't give a ton to go on. Also paste some code if you can.

**Comment by Benoye** [comment]
Hi sorry, it took me some time to get some logs !
Thank you for your response :)

Here is what I get :

```
***** Fatal JavaScript exception - application has been terminated. *****
NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'value' of null
at
(file: node_modules/@nstudio/nativescript-plaid/index.ios.js:100:0)
*** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'value' of null
at
(file: node_modules/@nstudio/nativescript-plaid/index.ios.js:100:0)
', reason: '(null)'
```

I have just implemented the code from the example, with no significant change..
Looking at the line, it appears that 'verificationStatus: account.verificationStatus' is null

I am going to check on that : I am using the "sandbox" mode where dummy accounts are used, maybe it is the problem.

**Comment by Benoye** [comment]
Ok so there is a need for a check on the `verificationStatus` to see if it is null or not
And there is a typo on lines 111 and 112 for the word `institution` : a "t" is missing !

Then it works great ! This plugin is going to be so useful for me !

**Comment by NathanWalker** [comment]
@Benoye are you still have issue here? Looking at the typings for the plugin, the mispelling is actually on the Plaid side (their SDK contains that mispelling `insitution` but according the the metadata it should be accurate and be fine).

I did post this the PR referenced as should help you other issue, lemme know if you believe that could help you and we can merge and get a patch out. I also bumped to use the latest Plaid SDK and could publish a 1.1.0 on this.

**Comment by Benoye** [comment]
Hi @NathanWalker, thanks for the fix ! I looked at the pull request, all good !

Regarding the mispelling, it was fixed in the 2.0.11 version of the LinkKit (https://github.com/plaid/plaid-link-ios/releases)
so if we upgrade to the latest Plaid SDK, maybe we should fix it as well ?

**Comment by NathanWalker** [comment]
Ok nice, lemme go ahead and publish 1.1.0 with the bumped pod to latest Plain 2.1.0 and will have your fix there, one moment.

**Comment by NathanWalker** [comment]
@Benoye this is published with 1.1.0 now, thanks again for the report.
